db/migrate/20250107204800_create_suggested_resources.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSuggestedResources < ActiveRecord::Migration[7.1]
```

Comment:
@jazairi I'm not sure how important this is, but the order of migrations is currently incorrect. I noticed this when trying to rollback the latest migration to get back to `main` and it rolled back 2 migrations that landed last week before getting to these 3.

I suspect it would be better to have these named such that they show up after the latest `main` migrations as I believe Rails just tracks the latest date migrations have completed and not each individual migration status so there is risk that some environments will skip these migrations and we'll be super confused.

I may be misunderstanding how this works though...

Reply:
Gotcha. Is that as simple as changing the date marker to, say, today?

Reply:
I think so 🫰

Reply:
(GitHub has the worst fingers crossed emoji. That's what that is supposed to be)